guideline/cfh.md

```diff
@@ -83,8 +85,7 @@ These are the requirements that an on-site team will sign-up to when submitting
 a proposal. Changes to these requirements are possible, but they must be signed
 off by the PAC before they can be put in place.
 
-1. The conference will be financially and legally run by an entity registered
```

Comment:
I think being or being affiliated with a legal entity is important.  This slightly raises the bar of holding PyCon APAC, but it looks a reasonable constraint to me.  It's not an overly difficult criteria, but it provides a clear signal that anyone who wants to hold PyCon APAC should establish a sound user base in the hosting country.

Also most if not all of the local PyCons started small and grew, I guess we all started with a certain kind of legal entity.  Be it a commercial company or a NPO.  Otherwise it's difficult to deal with authorities or sign contracts.